notebooks/2_1_MS_Windows_Studio_Effects.ipynb

```diff
@@ -13,6 +13,10 @@
     "\n",
     "* Demonstrate the NPU in action using the Windows Studio Effects\n",
     "\n",
+    "## Notes\n",
```

Comment:
Can you put this in one of the alert box? There's one below

Reply:
Sure, but I wasn't sure if we were avoiding the alert boxes because of potential issues with JupyterBook?

Reply:
We should not avoid them, they are working

Reply:
Okay cool, do you think this looks okay, or a little excessive? 
![image](https://github.com/STFleming/Riallto/assets/821289/4284670d-0241-41b9-81b4-252e45624ab0)

Reply:
More like this? 
![image](https://github.com/STFleming/Riallto/assets/821289/ef097d35-3035-45cc-a333-d16eb1eea2f0)

Reply:
I like the second option better